webapp/src/container.tsx

```diff
@@ -282,8 +282,8 @@ export class SettingsMenu extends data.Component<SettingsMenuProps, SettingsMenu
         const showCenterDivider = targetTheme.selectLanguage || targetTheme.highContrast || showGreenScreen || githubUser;
 
         return <sui.DropdownMenu role="menuitem" icon={'setting large'} title={lf("More...")} className="item icon more-dropdown-menuitem">
-            {showHome && <sui.Item className="mobile only inherit" role="menuitem" icon="home" text={lf("Home")} ariaLabel={lf("Home screen")} onClick={this.showExitAndSaveDialog} />}
-            {showShare && <sui.Item className="mobile only inherit" role="menuitem" icon="share alternate" text={lf("Share")} ariaLabel={lf("Share Project")} onClick={this.showShareDialog} />}
+            {showHome && <sui.Item className="mobile only inherit" role="menuitem" icon="home" title={lf("Home")} text={lf("Home")} ariaLabel={lf("Home screen")} onClick={this.showExitAndSaveDialog} />}
+            {showShare && <sui.Item className="mobile only inherit" role="menuitem" icon="share alternate" title={lf("Share/Publish")} text={lf("Share/Publish")} ariaLabel={lf("Share Project")} onClick={this.showShareDialog} />}
```

Comment:
With the text showing up in the dropdown menu having `Share/Publish` here feels bad to me; having a `/ synonym` on a button would make me think it leads to two options with a subtle difference between them and look a bit unfinished. Maybe we can have the title as something even more descriptive like 'Publish your game to create a shareable link' or similar and just leave the button as 'share'?